README.md

```diff
@@ -297,7 +396,7 @@ within the spec_helper RSpec configure:
 
 
 ## <a name="known-issues"></a>Known Issues
-BE/FE Integration of OAuth through GitHUb and authorization with JWT is in development.
+An admin is unable to create a mentor in the database. A mentor must log in and create the user by filling out the mentor form.
```

Comment:
I would like to phrase this positively and state our thinking behind this design decision. Can we say, "Admin cannot currently POST a new mentor. To POST, a prospective mentor logs in with GitHub, their Mentor Match User account is created through OAuth, and their mentor intake form data is then associated with that User. A future iteration could include a feature allowing an Admin to POST a new mentor independent of a User."